src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Dataflow/MethodBodyScanner.cs

```diff
@@ -482,8 +482,8 @@ protected virtual void Scan(MethodIL methodBody, ref InterproceduralState interp
                             if (methodBody.GetObject(reader.ReadILToken()) is MethodDesc methodOperand)
                             {
                                 TrackNestedFunctionReference(methodOperand, ref interproceduralState);
+                                HandleCall(methodBody, methodOperand, opcode, offset, new ValueNodeList() { MultiValueLattice.Top }, out MultiValue _);
```

Comment:
This doesn't match MethodBodyScanner.cs under ReferenceSource (or in the linker repo main):

https://github.com/dotnet/linker/blob/4db6ac94ffa8bf6dae8ad9c5c68f94b58de917fd/src/linker/Linker.Dataflow/MethodBodyScanner.cs#L391-L394

Do we understand why linker doesn't need it and are we comfortable with a diff? Cc @vitek-karas